.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
   - '2.7'
-  - '3.5'
   - '3.6'
+  - '3.7'
```

Comment:
nice. could you add py '3.8' ?

Reply:
We can not as long as `cffi`, one development dependency we have supports it as well.